server/paypal.js

```diff
@@ -0,0 +1,89 @@
+// Example:
+
+// url: https://api-m.sandbox.paypal.com/v1/payments/payouts
+
+// Authorization: "Bearer {token}"
+
+// {
+//   "sender_batch_header": {
+//     "sender_batch_id": "Payouts_2018_100007",
+//     "email_subject": "You have a payout!",
+//     "email_message": "You have received a payout! Thanks for using our service!"
+//   },
+//   "items": [
+//     {
+//       "recipient_type": "EMAIL",
+//       "amount": {
+//         "value": "1.1",
+//         "currency": "USD"
+//       },
+//       "note": "Thanks for your patronage!",
+//       "receiver": "sb-dmus17722352@personal.example.com"
+//     }
+//   ]
+// }
+const axios = require('axios');
+
+const paypal = async (orderToTransfer, payeeEmail, cb) => {
+  // Prepare the transacation order
+
+  const orderToTransferWithPayee = [];
+  orderToTransfer.forEach(order => {
+    const result = {};
+    result.recipient_type = 'EMAIL';
+    result.amount = {};
+    result.amount.value = order.amount.value;
+    result.amount.currency = order.amount.currency_code;
+    result.receiver = payeeEmail;
+    orderToTransferWithPayee.push(result);
+  });
+
+  console.log(orderToTransferWithPayee);
```

Comment:
We should remove these logs to clean up our logging/output

Reply:
Sure, I will remove them after all is good. Still have things to set up ATM